internal/addon/manifests/charts/mcoa/charts/metrics/templates/clusterrolebinding.yaml

```
@@ -0,0 +1,14 @@
+{{- if .Values.enabled }}
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: open-cluster-management:metrics-addon:agent
```

We should probably default to a naming similar to this one, I know the role+rolebinding that we already have there are using the naming you used but I feel it will make things clearer if we go with
`addon-name:signal-name:component`. WDYT?
```suggestion
  name: multicluster-observability-addon:metrics:agent
```